api/billingd/pb/billingd.proto

```diff
@@ -125,10 +137,10 @@ service APIService {
     rpc CheckHealth(CheckHealthRequest) returns (CheckHealthResponse) {}
     rpc CreateCustomer(CreateCustomerRequest) returns (CreateCustomerResponse) {}
     rpc GetCustomer(GetCustomerRequest) returns (GetCustomerResponse) {}
+    rpc ListDependentCustomers(ListDependentCustomersRequest) returns (ListDependentCustomersResponse) {}
```

Comment:
Hmmm, intersting. This just sparked a question: Because this is quite nice and generic (ref customers by keys), could we essentially have _any_ user be a parent of _any_ other user(s). This sounds like it'd support things like "family plans" (or similar). Just wondering out loud here.

Reply:
Yep, for sure doable!